src/easylink/rule.py

```diff
@@ -169,18 +171,19 @@ class InputValidationRule(Rule):
     """
 
     name: str
+    slot_name: str
     input: List[str]
     output: str
     validator: Callable
 
     def _build_rule(self) -> str:
         return f"""
 rule:
-    name: "{self.name}_validator"
+    name: "{self.name}_{self.slot_name}_validator"
```

Comment:
we might have duplicate slot names. I figured the best way to disambiguate is to do {node_name}_{slot_name}, which looks a little odd right now because all the slot names contain the step names. We should probably take the step names out of the slot names and just call them e.g. "main_input" and "secondary_input" or even just "main" and "secondary"

Reply:
> We should probably take the step names out of the slot names

Agreed!